docs/sidebars.ts

```diff
@@ -13,21 +13,464 @@ import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
+        },
+        {
+          type: 'doc',
+          id: 'recipes/index'
+        }
+      ]
+    },
+    {
+      type: 'category',
+      label: 'Evaluate & Monitor 📊',
+      className: 'sidebar-top-level-category',
+      collapsed: false,
+      items: [
+        {
+          type: 'doc',
+          id: 'model-evaluation/index',
+          label: 'MLflow Evaluation',
+        },
+        {
+          type: 'category',
+          label: 'MLflow Tracing (Observability)',
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'tracing',
+            },
+          ],
+          link: {
+            type: 'doc',
+            id: 'tracing/index',
+          }
+        },
+        {
+          type: 'doc',
+          id: 'dataset/index',
+          label: 'MLflow Dataset',
+        },
+      ]
+    },
+    {
+      type: 'category',
+      label: 'Deploy 🚀',
+      className: 'sidebar-top-level-category',
+      collapsed: false,
+      items: [
+        {
+          type: 'doc',
+          id: 'model-registry/index',
+        },
+        {
+          type: 'category',
+          label: 'MLflow Serving',
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'deployment',
+            }
+          ]
+        },
+        {
+          type: 'doc',
+          id: 'llms/deployments/index',
+          label: 'MLflow AI Gateway',
+        },
+      ]
+    },
+    {
+      type: 'category',
+      label: 'Team Collaboration 👥',
+      className: 'sidebar-top-level-category',
+      collapsed: true,
+      items: [
+        {
+          type: 'link',
+          href: '/tracking/#tracking-setup',
+          label: 'Self-Hosting'
+        },
+        {
+          type: 'link',
+          href: '/#running-mlflow-anywhere',
+          label: 'Managed Services'
+        },
+        {
+          type: 'doc',
+          id: 'auth/index',
+          label: 'Access Control',
+        },
+        {
+          type: 'doc',
+          id: 'projects/index',
+          label: 'MLflow Projects',
+        },
+      ]
+    },
+    {
+      type: 'category',
+      label: 'API References',
+      className: 'sidebar-top-level-category',
+      collapsed: true,
+      items: [
+        {
+          type: 'link',
+          label: 'Python API',
+          href: 'https://mlflow.org/docs/latest/python_api/index.html',
+        },
+        {
+          type: 'link',
+          label: 'Java API',
+          href: 'https://mlflow.org/docs/latest/java_api/index.html',
+        },
+        {
+          type: 'link',
+          label: 'R API',
+          href: 'https://mlflow.org/docs/latest/R-api/index.html',
+        },
+        {
+          type: 'link',
+          label: 'REST API',
+          href: 'https://mlflow.org/docs/latest/rest-api/index.html',
+        },
+        {
+          type: 'link',
+          label: 'CLI',
+          href: 'https://mlflow.org/docs/latest/cli.html',
```

Comment:
these should have `/api_reference`

```suggestion
        {
          type: 'link',
          label: 'Python API',
          href: 'https://mlflow.org/docs/latest/api_reference/python_api/index.html',
        },
        {
          type: 'link',
          label: 'Java API',
          href: 'https://mlflow.org/docs/latest/api_reference/java_api/index.html',
        },
        {
          type: 'link',
          label: 'R API',
          href: 'https://mlflow.org/docs/latest/api_reference/R-api/index.html',
        },
        {
          type: 'link',
          label: 'REST API',
          href: 'https://mlflow.org/docs/latest/api_reference/rest-api/index.html',
        },
        {
          type: 'link',
          label: 'CLI',
          href: 'https://mlflow.org/docs/latest/api_reference/cli.html',
```